core/src/main/java/org/springframework/ldap/core/support/AbstractContextSource.java

```diff
@@ -93,13 +93,13 @@
 	 * @deprecated use {@link #getUserDn()} and {@link #setUserDn(String)} instead
 	 */
 	@Deprecated
-	protected String userDn = "";
+	private String userDn = "";
```

Comment:
I think we need to leave this as protected so we don't break existing users. This is not a major release, so we should try to remain passive